CUDA is available: True
Using 5 GPUs for model parallelization.
Traceback (most recent call last):
  File "/home/usuaris/imatge/joan.manel.cardenas/age_predictions/input_data.py", line 113, in <module>
    x_sample = x[i].detach().numpy().reshape(-1)
IndexError: index 1 is out of bounds for dimension 0 with size 1